components/datetime/src/combo.rs

```diff
@@ -52,7 +52,7 @@ use crate::{provider::neo::*, scaffold::*};
 /// Same thing with a fixed calendar formatter:
 ///
 /// ```
-/// use icu::calendar::Gregorian;
+/// use icu_calendar::Gregorian;
```

Comment:
also everywhere else

```suggestion
/// use icu::calendar::Gregorian;
```